src/create-entity-api.ts

```diff
@@ -115,6 +153,14 @@ export function createEntityApi<
               : [entityName]
         }),
 
+        /**
+         * Creates a query endpoint for infinite searching entities. Behaves similar to `search`, but accumulates data from newly requested pages.
```

Comment:
It's better to write right here how the endpoint works so that users don't have to go to the documentation of `search`